traffic_ops/install/bin/_postinstall

```diff
@@ -711,10 +711,10 @@ def unmarshal_config(dct): # type: (dict) -> dict[str, list[Question]]
 
 	return ret
 
-def setup_maxmind(maxmind_answer, root): # type: (str, str) -> None
+def setup_maxmind(maxmind_answer, root, download_tool = '/usr/bin/curl'): # type: (str, str, str) -> None
```

Comment:
Changing the tool to use `curl` instead of `wget` is technically a breaking change in the otherwise "drop-in" script. The problem *should* be solved, as I understand it, by just `except`ing the proper type.

Reply:
The OSes that this project officially supports are CentOS 7 and CentOS 8. Neither OS includes `wget` by default and both include `curl` by default, so I consider the old tool using `wget` instead of `curl` a bug by itself.